guides/source/developers/api/orders.html.md

```diff
@@ -0,0 +1,477 @@
+# Orders
+
+Orders through the API will only be visible to admins and the users who own
+them. If a user attempts to access an order that does not belong to them, they
+will be met with an authorization error.
+
+## Cancel
+
+To cancel an order, make a request using that order's number:
+
+```text
+PUT /api/orders/R123456789
+```
+
+### Successful response
+
+**Response code** 200
+
+```json
```

Comment:
One small question: is this going to render as it is on the guides? If so, is there a way we can collapse it?

Reply:
Yes, as it stands it will render. I'll see if I can add a collapse.